### ISSUE TITLE
JSROOT.require("geom") returns emtry dict under nodejs

### ISSUE THREAD

**Issue by DraTeots** [root-project/jsroot#228]
I'm tryting to get a THREEJS object to export it to GLTF format. But when I run JSROOT.require("geom") to build a geometry it returns just a `{}`. 

Here is the full code to reproduce: 

```js
let jsroot = require("jsroot");
let fs = require("fs");

jsroot.openFile("https://eicweb.phy.anl.gov/EIC/detectors/athena/-/jobs/559705/artifacts/raw/geo/calorimeters_geo.root?inline=false")
      .then(file => file.readObject("default"))
      .then(eicGeometry => {
            var opt = { numfaces: 5000000, numnodes: 50000, dflt_colors: true, vislevel: 4 };
              jsroot.require('geom')
                .then(geo => {
                      console.log(geo);                           // <= geo is emtpy
                      let obj3d = geo.build(eicGeometry, opt);    // certainly it fails here
                      console.log(obj3d);                         // geometry export supposed to be here
                });
      })
```

Guess I'm doing something wrong here?

**Comment by DraTeots** [comment]
I also see that require("geom") winds up like this: 

```
[ 'geom' ]
[ 'd3', 'three', 'geobase', 'painter', 'base3d' ]
[ 'three', 'csg' ]
[ 'three' ]
[ 'd3' ]
[ 'd3', 'threejs_jsroot', 'painter' ]
[ 'three' ]
undefined
```

**Comment by linev** [comment]
It was a bug, now should be fixed in `master` and `6.3` branch
